cdk/database/__init__.py

```diff
@@ -14,7 +14,7 @@ def __init__(self, scope: core.Construct,
         super().__init__(
             scope, self.id, env=env, termination_protection=True)
 
-        self.dynamodb_single_table()  # This is the original table.
+        self.dynamodb_single_table()
```

Comment:
I think there is value in keeping this commented incase we arent able to clean up this code by the end of the semester